tests/test_other.py

```diff
@@ -4479,6 +4479,10 @@ def test_strftime_zZ(self):
     Popen([PYTHON, EMCC, 'src.cpp']).communicate()
     self.assertContained('ok!', run_js('a.out.js'))
 
+  def test_strptime_symmetry(self):
+    Building.emcc(path_from_root('tests','strptime_symmetry.cpp'), [], output_filename='a.out.js')
+    self.assertContained('TEST PASSED', run_process(JS_ENGINES[0] + ['a.out.js'], stdout=PIPE, stderr=PIPE).stdout)    
```

Comment:
Thanks, lgtm. Just some minor stuff for the test:

 * can remove the `[]` on the Building line, it's the default anyhow, and you use a keyword argument after it (so number of arguments doesn't matter).
 * the `run_process` stuff on the second line can be replaced with `run_js('a.out.js')`.